app/components/public/add-to-calender.ts

```diff
@@ -49,7 +49,7 @@ export default class AddToCalender extends Component<Args> {
     const { event } = this.args;
     const startTime = this.startsAt.utc().format('YYYY[-]MM[-]DDTHH[:]mm[:]SS[Z]');
     const endTime = this.endsAt.utc().format('YYYY[-]MM[-]DDTHH[:]mm[:]SS[Z]');
-    return `https://outlook.live.com/calendar/0/deeplink/compose?path=/calendar/action/compose&rru=addevent&subject=${event.name}&startdt=${startTime}&enddt=${endTime}&body=${event.description}&location=${this.args.location}`;
+    return `https://outlook.live.com/calendar/0/deeplink/compose?subject=${event.name}&startdt=${startTime}&enddt=${endTime}&body=${(event.description).substring(0,80)}&location=${this.args.location}`;
```

Comment:
80 is quite small, is it max?

Reply:
It's no where documented, I will have to try and test what is the limit.

Reply:
Also event name and description show + instead of space. Check if there is an easy fix, otherwise it is fine